bitrise.yml

```diff
@@ -83,6 +83,7 @@ workflows:
         - scheme: Datadog
         - simulator_device: iPhone 11
         - is_clean_build: 'yes'
+        - should_retry_test_on_fail: 'yes'
```

Comment:
a comment here would be good as this is a temporary fix

Reply:
```suggestion
        - should_retry_test_on_fail: 'yes' # temporarily mutes flakiness until we collect more info (in RUMM-839) then fix it
```